doc/JSON_INFO.md

```diff
@@ -2388,6 +2388,31 @@ Alternately, every item (book, tool, gun, even food) can be used as armor if it
     "power_armor" : false
 }
 ```
+Guidelines for thickness.
+According to https://propercloth.com/reference/fabric-thickness-weight/
+Dress shirts and similar fine clothing ranges from 0.15mm to 0.35mm.
+According to https://leathersupreme.com/leather-hide-thickness-in-leather-jackets/
+Fashion leather such as thin leather jackets, skirts, and thin vests are 1.0mm or less.
+Heavy leather clothes such as motorcycle suits average 1.5mm.
+According to http://cci.one/site/marine/design-tips-fabrication-overview/tables-of-weights-and-measures/?doing_wp_cron=1611725817.5850219726562500000000
+An equivalency guideline for fabric weight to mm.
+2.3 oz/yd2 fiberglass (plain weave cloth) 	78 gm/m2 	0.004″ 	0.10 mm
+6.0 oz/yd2 fiberglass (plain weave cloth) 	203 gm/m2 	0.007″ 	0.17 mm
+5.0 oz/yd2 Kevlar (TM) (plain weave cloth) 	170 gm/m2 	0.010″ 	0.25 mm
+5.8 oz/yd2 Carbon Fiber (plain weave cloth) 	197 gm/m2 	0.009″ 	0.23 mm
+9.0 oz/yd2 Carbon Fiber (unidirectional) 	305 gm/m2 	0.011″ 	0.28 mm
+
+Chart cobbled together from several sources
+Very lightweight fabrics are between 0-4oz. (up to 136 gsm) 0.006″-0.007″ 0.15mm
+Light fabrics are between 4-7oz. (up to 237 gsm) 0.008″ 0.2mm
+Medium fabrics are between 7-11oz. (up to 373 gsm) 0.009″-0.011" 0.25mm
+Heavy fabrics are between 11-14oz. (up to 475 gsm) 0.012″-0.014″ 0.3mm
+
+Shoe thicknesses outlined at https://secretcobbler.com/choosing-leather/
+TL;DR upper 1.2 - 2.0mm, lining 0.8 - 1.2mm, for a total of 2.0 - 3.2mm
+
+Reference for turnout gear http://bolivar.mo.us/media/uploads/2014/09/2014-06-bid-fire-gear-packet.pdf
```

Comment:
```suggestion
#### Guidelines for thickness: ####
According to [this site](https://propercloth.com/reference/fabric-thickness-weight/), dress shirts and similar fine clothing range from 0.15mm to 0.35mm.
According to [this site](https://leathersupreme.com/leather-hide-thickness-in-leather-jackets/):
* Fashion leather such as thin leather jackets, skirts, and thin vests are 1.0mm or less.
* Heavy leather clothes such as motorcycle suits average 1.5mm.
From [this site](http://cci.one/site/marine/design-tips-fabrication-overview/tables-of-weights-and-measures/?doing_wp_cron=1611725817.5850219726562500000000), an equivalency guideline for fabric weight to mm:

| Cloth | g/m2 | Inches | mm |
| ----- | ------ | ------ | ---- |
| 2.3 oz/yd2 fiberglass (plain weave cloth) | 78 | 0.004 | 0.10 |
| 6.0 oz/yd2 fiberglass (plain weave cloth) | 203 | 0.007 | 0.17 |
| 5.0 oz/yd2 Kevlar (TM) (plain weave cloth) | 170 | 0.010 | 0.25 |
| 5.8 oz/yd2 Carbon Fiber (plain weave cloth) | 197 | 0.009 | 0.23 |
| 9.0 oz/yd2 Carbon Fiber (unidirectional)  | 305 | 0.011 | 0.28 |

Chart cobbled together from several sources:
* Very lightweight fabrics are 0-4oz. (up to 136 gsm): 0.006″-0.007″ - use 0.15mm
* Light fabrics are 4-7oz. (up to 237 gsm): 0.008″ - use 0.2mm
* Medium fabrics are 7-11oz. (up to 373 gsm): 0.009″-0.011" - use 0.25mm
* Heavy fabrics are 11-14oz. (up to 475 gsm): 0.012″-0.014″ - use 0.3mm

Shoe thicknesses are outlined at [this site](https://secretcobbler.com/choosing-leather/); TL;DR: upper 1.2 - 2.0mm, lining 0.8 - 1.2mm, for a total of 2.0 - 3.2mm

For turnout gear, see [this reference](http://bolivar.mo.us/media/uploads/2014/09/2014-06-bid-fire-gear-packet.pdf).
```
This isn't quite finished - I should probably turn the last listing into a table also - but it's a start.

Reply:
Thanks a ton! :1st_place_medal:

Reply:
Quite welcome; happy to help!